### DISCUSSION TITLE
希望能定制看板娘的位置

### DISCUSSION THREAD

**Discussion by HeavySnowJakarta** [stevenjoezhang/live2d-widget#95] [Ideas]
看板娘的位置默认是左下角，但是我的个人博客侧边栏在右边，看板娘在左边把网页底部的字都挡住了，我希望能提供把看板娘移到右下角的参数。

**Comment by PFiS1737** [comment]
可以参考 https://github.com/stevenjoezhang/live2d-widget/pull/64#issue-808127313

**Comment by stevenjoezhang** [comment]
感谢建议！已增加拖动功能： https://github.com/stevenjoezhang/live2d-widget/commit/aae552759611e205fb8d5a49f0349f2630bc0d1c
